src/Module/Photo.php

```diff
@@ -182,6 +182,12 @@ protected function rawContent(array $request = [])
 			throw new HTTPException\InternalServerErrorException($error);
 		}
 
+		if (!empty($request['static'])) {
+			$img = new Image($imgdata, $photo['type']);
+			$img->toStatic();
+			$imgdata = $img->asString();
```

Comment:
This should be cached.

Reply:
Caching is not needed here, since we are sending the appropriate headers, so that the browser will perform the caching.